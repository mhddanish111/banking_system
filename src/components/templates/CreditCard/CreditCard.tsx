import React from 'react';
import Layout from '../Layout';
import AccountTransaction from '../../organisms/AccountTransaction';

const CreditCard: React.FC<any> = () => {
  return (
    <Layout pageHeading="Credit Card">
      <AccountTransaction transactionType="Cards" />
    </Layout>
  );
};

export default CreditCard;
