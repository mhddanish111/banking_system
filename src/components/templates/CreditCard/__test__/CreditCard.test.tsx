import { shallow } from 'enzyme';
import CreditCard from '../CreditCard';

describe('<CreditCard />', () => {
  const wrapped = shallow(<CreditCard />);
  it('should render the CreditCard Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
