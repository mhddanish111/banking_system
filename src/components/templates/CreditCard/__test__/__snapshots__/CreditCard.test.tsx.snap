// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreditCard /> should render the CreditCard Component correctly 1`] = `
<Layout
  pageHeading="Credit Card"
>
  <AccountTransaction
    transactionType="Cards"
  />
</Layout>
`;
